README.md

```diff
@@ -0,0 +1,88 @@
+# GameOfLife README
+
+# Description
+
+GameOfLife is a Java simulation of the famous Conway's Game of Life.
+It consists primarily of a grid made of clickable cells.
+Cells can be alive, in which case they will light up with a color, or dead.
+Cell neighbors are adjacent or immediately diagonal cells. Depending on the
+first configuration of living and dead cells, the cells that live and die in
+the next generation will depend on the four basic rules of the game:
+
+1. A live cell with 2 or 3 neighbors continues to live into the next generation.
+2. A live cell with 1 or 0 neighbors will die of loneliness in the next generation.
+3. A live cell with 4+ neighbors will die of overcrowding in the next generation.
+4. A dead cell with exactly 3 neighbors comes alive in the next generation.
+
+# Structure
+
+This program is comprised of 3 primary packages:
```

Comment:
In general, this amount of documentation is too specific and unhelpful. 
Also very hard to keep up to date when coffee changes.

Reply:
Ok... only ever wrote README files for my internship in the summer and they wanted them super thorough like this, so I am not sure how much info you want. Do you have a sample of how you usually want REAMEs to look?

Reply:
You don't need to change anything right now. But for the future you don't need to be this detailed.